pages/portfolio.html

```diff
@@ -1,12 +1,100 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="../stylesheets/portfolio.css">
 </head>
+
 <body>
-  
+  <header>
+    <h1>PORTFOLIO</h1>
+    <nav>
+      <ul>
+        <li><a href="../index.html">Home</a></li>
+        <li><a href="about.html">About</a></li>
+        <li><a href="portfolio.html">Portfolio</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <section class="project">
+      <img class="project_img" src="../img/portfolio/aquarium.png"
+          alt="Parents and kids looking at fish at an aquarium.">
+      <div class="project_text">
+        <h2 class="title">Landing Page</h2>
+        <p class="description">A landing page of a fictional nonprofit called Mango Island Aquarium.</p>
+        <p class="project_language">HTML / CSS</p>
+        <a href="https://github.com/anacapamu/odin-landing_page">GitHub</a> /
+        <a href="https://anacapamu.github.io/odin-landing_page/">Demo</a>
```

Comment:
for links you can use the target attribute to open the links in a new tab
```suggestion
        <a href="https://github.com/anacapamu/odin-landing_page" target="_blank">GitHub</a> /
        <a href="https://anacapamu.github.io/odin-landing_page/"  target="_blank" >Demo</a>
```